nmma/em/model.py

```diff
@@ -291,7 +291,7 @@ def __init__(
                 if filters is None:
                     self.filters = model_filters
 
-            if os.path.isfile(modelfile):
+            elif os.path.isfile(modelfile):
```

Comment:
This if statement is meant to stand on its own. Maybe the local_only flag is not doing what you want?

Reply:
No, i think the local_only flag is alright, i think. This was just a quick fix from my side, where I use the Bu2022Ye model. 
I wanted to prevent it from jumping in to the else statement below, where it tries to read in the _mag_tf model, which doesn't exist for Bu2022Ye. 

As i said this is rather hacky and now that i look at it again probably inappropriate. I think the issue is that for Bu2022Ye os.path.isfile(modelfile) is False, because modelfile actually does not point to Bu2022Ye_tf.pkl but rather to Bu2022Ye.pkl which does not exist. But I don't know how to fix this consistently for the entirety of the models out there.

Reply:
@haukekoehn If `_tf.pkl` is the problem, maybe try stripping off the `_tf` from `Bu2022Ye_tf.pkl` and use then use it with `--local-only`

Reply:
i did it the other way around, where now `_tf` is added to the `modelfile` variable, when the interpolation type is tensorflow. Maybe we can chat tomorrow about this.